src/Metable.php

```diff
@@ -41,6 +41,32 @@ public static function bootMetable()
         });
     }
 
+    /**
+     * Override: Get attribute from model, then meta
+     *
+     * @param string $key
+     * @return mixed
+     */
+    public function getAttribute($key)
+    {
+        return parent::getAttribute($key) ?? $this->getMeta($key);
+    }
+
+    /**
+     * Helper to get the default method value if it's set
+     *
+     * @param string $key
+     * @return void
+     */
+    protected function getDefaultMetaValueByKey($key)
+    {
+        if (isset($this->defaultMetaValues) && array_key_exists($key, $this->defaultMetaValues)) {
+            return $this->defaultMetaValues[$key];
+        }
```

Comment:
`property_exists()` would be required here, given the (very unlikely) scenario that there is an actual attribute by that name